persistent-volumes/efs/double_containers.json

```diff
@@ -0,0 +1,73 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "doublecontainers",
+    "creationTimestamp": null,
+    "labels": {
+      "name": "hello-openshift"
+    }
+  },
+  "spec": {
+    "containers": [
+      {
+        "name": "hello-openshift",
+        "image": "jhou/hello-openshift",
```

Comment:
typically we will want to avoid using personal repos- can we use the aosqe/hello-openshift pod here?